.github/workflows/push.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Build
+on: [push]
+
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.6', '3.7']
```

Comment:
What's the point of linting with 2 different python version ?
Is it possible for a code to be found OK with 3.7 but dangerous/bogus with 3.6 ?

Reply:
Sorry @strk have only seen you have commented on this. 

This is following the recommendations of the [template-python-hello-world](https://github.com/linz/template-python-hello-world). This repo has been set up as a template for LINZ python projects. 

>What's the point of linting with 2 different python version ?
Is it possible for a code to be found OK with 3.7 but dangerous/bogus with 3.6 ?

Perhaps @dwsilk can comment?

Reply:
QGIS uses 3.6.8. TODO// Remove 3.7